doc/schema.md

```diff
@@ -29,3 +29,11 @@ In order to save storage space, we store the full transaction IDs once, and use
 
 Note that this mapping allows us to use `getrawtransaction` RPC to retrieve actual transaction data from without `-txindex` enabled
 (by explicitly specifying the [blockhash](https://github.com/bitcoin/bitcoin/commit/497d0e014cc79d46531d570e74e4aeae72db602d)).
+
```

Comment:
nit and maybe OT for this PR but should we use BU commitid rather than Core?  
i.e. https://github.com/BitcoinUnlimited/BitcoinUnlimited/commit/a41b8d26bc1807a629f995b87bd2ce5d3692465f

Reply:
I have not made any effort into rebranding. IMO rather than linking to a commit, I think it's better to link to BU RPC documentation, or not link at all.

Reply:
fair enough, Agreed.